ecosystem/sep-0024.md

```diff
@@ -12,7 +12,7 @@ Version 1.0.0
 
 ## Simple Summary
 
-This SEP defines the standard way for anchors and wallets to interact on behalf of users. This improves user experience by allowing wallets and other clients to interact with anchors directly without the user needing to leave the wallet to go to the anchor's site.  It is based on, and backwards compatible with, [SEP-0006](sep-0006.md), but only supports the interactive flow, and cleans up or removes confusing artifacts.
+This SEP defines the standard way for anchors and wallets to interact on behalf of users. This improves user experience by allowing wallets and other clients to interact with anchors directly without the user needing to leave the wallet to go to the anchor's site.  It is based on, and backwards compatible with, [SEP-0006](sep-0006.md), but only supports the interactive flow, and cleans up or removes confusing artifacts.  If you are updating from SEP-0006 see the [changes from SEP-6](#changes-from-SEP-6) at the bottom of this document.
```

Comment:
This update has a conflict with my latest PR (#509). Maybe we can remove the `, and backwards compatible with` here and delete my PR?